### PR TITLE
Add users page and improve contact info

### PR DESCRIPTION
**Pull request by mosabua**
## Description

Small for now but a good start in prep for Trino Summit. I will add more and more info for a bit and then merge and get it live after approval. And then we can add more after as well.

## Release notes

(x) This is not user-visible or is docs only, and no release notes are required.

### CODE REVIEW

File: docs/users.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+## Bloomberg
+
+Maintainer vishal, contributor andy, initial massive contribution, "founding 
+company" for the project
+
+## Expedia
+
+Contributor of UI changes
+
+## Line
```
Comment (ebyhr) [inline docs/users.md:26]: The name should be [LY](https://www.lycorp.co.jp/en/). LINE and Yahoo! JAPAN has been merged into one.

File: docs/index.md [hunk 2]
```diff
@@ -21,7 +21,13 @@ gateway for multiple [Trino](https://trino.io) clusters.
 
 ## Other resources
```
Comment (mosabua) [inline docs/index.md:22]: Related question for reviewers .. should we lift this whole section into a top level "Resources" page .. similar to the users page and therefore keep the front page shorter?

**Review by Chaho12 — APPROVED** [review]
LGTM 👍

File: docs/users.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+Trino Gateway contributor [Isa Inalcik](https://github.com/alaturqua) works with
+Trino Gateway in the data platform at BestSecret as mentioned in
+[Trino Community Broadcast 60](https://www.youtube.com/watch?v=iAMOkhkg44A&t=852s)
+
+## [Bloomberg](https://www.bloomberg.com/)
+
+Trino Gateway maintainer [Vishal Jadhav](https://github.com/vishalya) and 
+contributor [Erik Anderson](https://github.com/dprophet) initiated the project
+with others. [Andy Su](https://github.com/andythsuœ) is also a contributor. A
```
Comment (andythsu) [inline docs/users.md:25]: there's a weird symbol at the end of my name 😆

Comment (andythsu) [reply docs/users.md:25]: should just be `https://github.com/andythsu`

File: docs/index.md (hunk 2 above)
Comment (mosabua) [reply docs/index.md]: Moved

Comment (mosabua) [reply docs/index.md]: Thanks for confirming my suspicion..

**Pull request merged**